pwiz_tools/Skyline/TestUtil/PythonTestUtil.cs

```diff
@@ -204,10 +218,10 @@ private void RunNvidiaDialog(MultiButtonMsgDlg nvidiaDlg)
         private void RunLongPathsDialog(MessageDlg longPathDlg)
         {
             Console.WriteLine(@"Info: Trying to set LongPathsEnabled registry key to 1");
-            OkDialog(longPathDlg, longPathDlg.OkDialog);
+            AbstractFunctionalTest.OkDialog(longPathDlg, longPathDlg.OkDialog);
             Console.WriteLine(@"Info: Successfully set LongPathsEnabled registry key to 1");
             _undoRegistry = true;
-            MessageDlg okDlg = WaitForOpenForm<MessageDlg>();
+            MessageDlg okDlg = AbstractFunctionalTest.WaitForOpenForm<MessageDlg>();
             okDlg.OkDialog();
```

Comment:
I don't think this is actually reachable code (unless running Skyline as admin, which would be unusual) but this line should be
`
            AbstractFunctionalTest.OkDialog(okDlg, okDlg.OkDialog);
`

Reply:
I wonder if there isn't a way we could let a user who does have sufficient rights to set that registry key - launch a tiny app that does only that, and only works if the user clicks through the OS-provided query about it